support/config/deployment_test.go

```diff
@@ -167,6 +167,52 @@ func TestSetControlPlaneIsolation(t *testing.T) {
 	}
 }
 
+func TestSetControlPlaneTolerations(t *testing.T) {
+	hcp := &hyperv1.HostedControlPlane{}
+	hcp.Name = "hcp-name"
+	hcp.Namespace = "hcp-namespace"
+	customTolerations := []corev1.Toleration{
+		{
+			Key:      "key1",
+			Operator: corev1.TolerationOpEqual,
+			Value:    "value1",
+			Effect:   corev1.TaintEffectNoSchedule,
+		},
+		{
+			Key:      "key2",
+			Operator: corev1.TolerationOpEqual,
+			Value:    "value2",
+			Effect:   corev1.TaintEffectNoSchedule,
+		},
+	}
+	hcp.Spec.Tolerations = append(hcp.Spec.Tolerations, customTolerations...)
+
+	cfg := &DeploymentConfig{}
+	cfg.setControlPlaneIsolation(hcp)
```

Comment:
I think you have now to call _`setAdditionalTolerations`_

Reply:
fixed, thanks